htdp-doc/scribblings/htdp-langs/common.rkt

```diff
@@ -16,7 +16,7 @@
 @; -----------------------------------------------------------------------------
 (define dots (bold "..."))
 
-(define htdp "http://www.ccs.neu.edu/home/matthias/HtDP2e/")
+(define htdp "https://htdp.org/2018-01-06/Book/")
```

Comment:
This points to an old release of "2e". Let's just point to HtDP.org please.

Reply:
apologies now https://htdp.org/2020-5-6/Book/